src/main/java/mekanism/common/config/StorageConfig.java

```diff
@@ -0,0 +1,82 @@
+package mekanism.common.config;
+
+import mekanism.common.config.options.DoubleOption;
+
+public class StorageConfig extends BaseConfig {
+
+    public final DoubleOption enrichmentChamberStorage = new DoubleOption(this, "storage",
```

Comment:
I don't think these need `Storage` in the field name (as they'll always be accessed via `usage.X`, probably don't need to be in the config key name either, but that's less of an issue as it does help context there

Reply:
should I also remove the Usage part in here?

```
public class UsageConfig extends BaseConfig {

    public final DoubleOption enrichmentChamberUsage = new DoubleOption(this, "usage", "EnrichmentChamberUsage", 50D,
          "Energy per operation tick (Joules).");
```

Reply:
yeah go for it